experimenter/experimenter/theme/static_src/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "theme",
+  "version": "3.6.0",
+  "description": "",
+  "scripts": {
+    "start": "npm run dev",
+    "build": "npm run build:clean && npm run build:tailwind",
+    "build:clean": "rimraf ../static/css/dist",
+    "build:tailwind": "cross-env NODE_ENV=production tailwindcss --postcss -i ./src/styles.css -o ../static/css/dist/styles.css --minify",
```

Comment:
We shouldn't need `cross-env`. We only support building our container on Linux and macOS, which support the `env` command.